src/init.cpp

```diff
@@ -427,17 +427,12 @@ static void registerSignalHandler(int signal, void(*handler)(int))
 }
 #endif
 
-static boost::signals2::connection rpc_notify_block_change_connection;
 static void OnRPCStarted()
 {
-    rpc_notify_block_change_connection = uiInterface.NotifyBlockTip_connect(std::bind(RPCNotifyBlockChange, std::placeholders::_2));
 }
 
 static void OnRPCStopped()
 {
-    rpc_notify_block_change_connection.disconnect();
-    RPCNotifyBlockChange(nullptr);
-    g_best_block_cv.notify_all();
```

Comment:
In commit "Replace RPCNotifyBlockChange with waitTipChanged()" (9d3b07526110a842c9592425b89785e688a8f46c)

g_best_block_cv still exists, so I don't think you should delete this line

Can move it to the place this function is called when this function is deleted in the next commit.